### ISSUE TITLE
Ignore dependencies via a config file?

### ISSUE THREAD

**Issue by molant** [dependabot/dependabot-core#828]
Hello all,

I'm evaluating switching to dependabot from another system and I have 2 questions I hope you can answer:

* Is it possible to tell dependabot to ignore a package ahead of time? We have a few dependencies that should never be updated but right now it seems the only way will have to wait until the PR is open to tell it to ignore. And even in that case I'm not sure if I won't get another message if there's a major bump later on.
* Is it possible to change the commit messages?

I've look for a config file to set in my repo but haven't found anything.

Thank you!

**Comment by paulmelnikow** [comment]
+1 on ignoring and unignoring dependencies via inspectable, mutable repo state.

**Comment by molant** [comment]
Thanks @greysteil for your answer!
Can't wait to see this implemented in the next few weeks 😊

> Yes! Dependabot commit messages aren't 100% configurable, but we try to accommodate everything users want. 

It will be swell if I could have a property in the configuration file similar to:

```json
{
  "commitMessage": "Chore: Update ${dependency} to v${version}"
}
```

What I'd like to have is a configuration file that I can drop in any of my projects and avoid having to set up anything via the user interface or commit messages. Is this in your roadmap?

**Comment by greysteil** [comment]
> What I'd like to have is a configuration file that I can drop in any of my projects and avoid having to set up anything via the user interface or commit messages. Is this in your roadmap?

Yep! Going to be the next big thing we ship - working on it right now. 🙂

On the commit message, that's pretty close to what you'll get if Dependabot detects you're using capitalised Angular prefixes without a scope. I think right now it will be `Chore: Bump {dep-name} from {old-v} to {new-v}`. Once we have config files that make it easier to write a store settings like fully custom commit messages I think it will make sense for Dependabot to support them, though.

**Comment by molant** [comment]
Awesome!
